code/modules/SCP/SCPs/SCP-173.dm

```diff
@@ -321,7 +301,150 @@ GLOBAL_LIST_EMPTY(scp173s)
 	var/area/A = get_area(src)
 	A.full_breach()
 
-// SCP-173 Cage
+// 173 AI procs
+
+/mob/living/scp_173/proc/handle_AI()
+	if(istype(loc, /obj/structure/scp173_cage))
+		loc.relaymove(src, NORTH)
+		return
+
+	var/list/possible_human_targets = list()
+
+	for(var/mob/living/carbon/human/H in dview(14, src)) //Identifies possible human targets. Range is double regular view to allow 173 to pursue tarets outside of world.view to make evading him harder.
+		if(H.SCP)
+			continue
+		if(H.stat == DEAD)
```

Comment:
Could prob combine these two into an OR. AStar one can stay separate since it's huge.